web/admin/src/RightReportsTable.js

```diff
@@ -47,21 +52,51 @@ export default class RightReportsTable extends Component {
                   returnContent={returnContent}
                   unBlock={approveQ}
                   getUser={getUser}
-                  report = {allReportsFlagged}
+                  report = {allReports}
                   content = {itemAndReport.item}
-                  singleReport = {allReportsFlagged[0]}
-                  allReportsFlagged = {allReportsFlagged}
+                  singleReport = {allReports[0]}
+                  allReportsFlagged = {allReports}
                 />
               </li>
             )
           }
         }) }
-        {(totalBlocked) ? null : this.renderMessage("Blocked items will appear here", "Any item in this list will not be", "visible to attendees")}
+        {this.renderMessageBox()}
       </ul>
     </div>
     )
   }
 
+  renderMessageBox = () => {
+    if (!this.state.isShowingApproved){
```

Comment:
There are some instances where no return statement is executed, (so `undefined` will be returned).  Explicitly return `null` at the end if this is intended.